foreman/foreman.spec

```diff
@@ -557,14 +559,32 @@ rm -rf %{buildroot}
 
 %files
```

Comment:
@lzap I'd appreciate your review of this verbose change... I think it's inevitable, please see the comment in the commit message.